variable/include/scipp/variable/transform.h

```diff
@@ -98,6 +99,67 @@ template <int N, class T> static constexpr auto get(const T &index) noexcept {
 
 } // namespace iter
 
+template <size_t N_Operands, bool in_place>
+inline constexpr auto stride_special_cases =
+    std::array<std::array<scipp::index, N_Operands>, 0>{};
+
+template <>
+inline constexpr auto stride_special_cases<1, true> =
+    std::array<std::array<scipp::index, 1>, 2>{{{1}}};
+
+template <>
+inline constexpr auto stride_special_cases<2, true> =
+    std::array<std::array<scipp::index, 2>, 4>{{{1, 1}, {0, 1}, {1, 0}}};
+
+template <>
+inline constexpr auto stride_special_cases<3, true> =
+    std::array<std::array<scipp::index, 3>, 3>{
+        {{1, 1, 1}, {1, 0, 1}, {1, 1, 0}}};
```

Comment:
Is this for functions with `out` args, or where is this used concretely?

Reply:
Strange, I don't remember adding that case...
Do we have binary operations with out args?

Reply:
None that I am aware of. -> remove for now?

Reply:
Done